helm-charts/konk-service/templates/deployment.yaml

```diff
@@ -62,7 +62,7 @@ spec:
       volumes:
       - name: kubeconfig
         secret:
-          secretName: {{ .Values.konk.name | required "must specify konk.name" }}-konk-kubeconfig
+          secretName: {{ .Values.konk.name | required "must specify konk.name" }}-kubeconfig
```

Comment:
I'm suspicious that this will work because I don't see a diff on the section that creates this secret.

Reply:
this should be calling the helper

Reply:
This will work. Since `-konk` suffix is added while creating the konk, the kubeconfig secret is named as `runner-konk-kubeconfig`.

Reply:
Adding a test